src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageServiceProtocol/LanguageServerTests.cs

```diff
@@ -1376,6 +1376,16 @@ public void CustomCheckResult()
             Assert.Contains("The type of this expression does not match the expected type 'Text'. Found type 'Decimal'.", diags.First().Message);
         }
 
+        [Fact]
+        public async Task TestExpectedReturnValueForEmptyExpression()
+        {
+            var scope = TestCreateEditorScope(string.Empty);
+
+            var result = scope.Check(string.Empty);
+
+            Assert.True(result.IsSuccess);
```

Comment:
what happens if we call result.GetEvaluator()  and try to run it? 
Does it succeed and return Blank()?